packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -505,8 +505,13 @@ const addSetImport = () => {
   const routesPath = getPaths().web.routes
   const routesContent = readFile(routesPath).toString()
   const [redwoodRouterImport, importStart, spacing, importContent, importEnd] =
-    routesContent.match(/(import {)(\s*)([^]*)(} from '@redwoodjs\/router')/) ||
+    routesContent.match(/(import {)(\s*)([^]*)(} from ['"]@redwoodjs\/router['"])/) ||
     []
+
+  if (!redwoodRouterImport) {
+    return "Couldn't add Set import from @redwoodjs/router to Routes.{js,tsx}"
+  }
```

Comment:
@zackdotcomputer it's great that we're handling errors here. One thing though that's no fault of yours (just our disorganized code) is that if you return from a listr task, the task is considered successful. I.e., there's a green check mark in your console (✅).

But if we return here, it wasn't really because the task was successful. It was actually skipped. Listr lets us skip tasks in the middle of their execution with the task object. You can read more about it here: https://www.npmjs.com/package/listr#task-object. Note that to get access to this task object, you'll have to change the function signature of `task: async () => addSetImport({ model, path })` a bit. Also not sure why we pass model and path when they're not being used, we definitely need to lean this up! 😅

Reply:
👍 Change made. Tested locally this failure case now shows:
```
 ↓ Adding set import... [skipped]
    → Couldn't add Set import from @redwoodjs/router to Routes.{js,tsx}
```
but finishes the rest of scaffolding gracefully

Reply:
Awesome!